app/controllers/appointments_controller.rb

```diff
@@ -38,6 +41,37 @@ def update
 
   private
 
+  def apply_search_filters
+    @patient_summaries = if @search_filters.include?("only_less_than_year_overdue")
+      policy_scope([:overdue_list, PatientSummary]).overdue
+    else
+      policy_scope([:overdue_list, PatientSummary]).all_overdue
+    end
+    if @search_filters.include?("phone_number")
+      @patient_summaries = @patient_summaries.where("latest_phone_number is not null")
+    end
+    if @search_filters.include?("no_phone_number")
+      @patient_summaries = @patient_summaries.where("latest_phone_number is null")
+    end
+    if @search_filters.include?("high_risk")
+      @patient_summaries = @patient_summaries.where("risk_level == 1")
+    end
+    @patient_summaries
+  end
+
+  def set_patient_search_filters
+    @patient_search_filters = {
+      "only_less_than_year_overdue" => "< 365 days overdue",
```

Comment:
This is awkward, need to ask about doing the inverse of this filter.

Reply:
Sure - I think we should wait till that terminology is encoded into the data dictionary to make that change, though. What do you think @harimohanraj89 @timcheadle ?

Reply:
Sounds like the sensible thing to do 👍 I'll resolve my other comments around this.